include/hipSYCL/common/appdb.hpp

```diff
@@ -29,7 +30,7 @@ struct kernel_arg_value_statistics {
   uint64_t last_used = 0; // The number of the kernel invocation where this value
                       // was last used
 
-  void dump(std::ostream& ostr, int indentation_level=0) const;
+  ACPP_COMMON_EXPORT void dump(std::ostream& ostr, int indentation_level=0) const;
```

Comment:
Note to future us: We may have to check whether the PCUDA runtime API that the AdaptiveCpp now exports also needs some Windows declspec export.

Reply:
This is already rebased merged in - and it didn't complain so far in the CI... so maybe not? :)
(but CI isn't doing anything specific to test PCUDA.. should it?)

Reply:
Looks like we currently only test PCUDA in self-hosted CI on Linux: https://github.com/AdaptiveCpp/AdaptiveCpp/blob/2a2795b73afb77530de2d01907bef22267adfd02/.github/workflows/linux-self-hosted.yml#L102